docker/Dockerfile

```diff
@@ -2,6 +2,8 @@
 FROM ubuntu 
 FROM python:3.7
 
+USER root
```

Comment:
Isn't this implicit?

Reply:
Yes it is..... :) 
This was from the era when I was turning every stone in order to make poetry work.